frontend/src/components/billing/components/replenish-cards-list/styles.module.scss

```diff
@@ -1,7 +1,6 @@
 .cardsWrapper {
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: center;
-  gap: 20px;
-  padding: 20px;
+  display: grid;
```

Comment:
Maybe it would be better to left it flex? On the screenshot it does not look very good :c

Reply:
I thought in that way too, but QAs want 2 rows 😖

Reply:
I meant 4 columns :)

Reply:
As for 4 cols grid - it would be better to center them 🤔